package.json

```diff
@@ -14,18 +14,23 @@
     "@openzeppelin/contracts": "^3.1.0"
   },
   "devDependencies": {
+    "@nomiclabs/hardhat-truffle5": "^2.0.0",
+    "@nomiclabs/hardhat-web3": "^2.0.0",
     "@openzeppelin/test-helpers": "^0.5.6",
     "bn-chai": "^1.0.1",
     "chai": "^4.2.0",
     "ethereumjs-abi": "^0.6.8",
+    "hardhat": "^2.1.1",
     "solhint": "^3.2.0",
-    "solidity-coverage": "^0.7.10",
-    "truffle": "^5.1.42"
+    "solidity-coverage": "^0.7.16",
+    "truffle": "^5.1.42",
+    "web3": "^1.3.4"
   },
   "scripts": {
-    "test": "truffle test",
+    "compile": "hardhat compile",
```

Comment:
I get the following warnings:
![image](https://user-images.githubusercontent.com/33000441/111456205-fedf2800-8716-11eb-99b9-9cc8e7f58ef8.png)

Reply:
license headers don't matter much. but we can add this in another pr if needed. changing the `balance` name would break contract abi which I would want to avoid.

Reply:
I think it would be good to add license headers. Added: https://github.com/ethersphere/swap-swear-and-swindle/issues/122